app/components/osc-webhook-triggers/osc-webhook-triggers.component.js

```diff
@@ -20,7 +21,7 @@
   });
 
 
-  function OscWebhookTriggers($scope, $uibModal, $filter, APIService) {
+  function OscWebhookTriggers($scope, $uibModal, $filter, $timeout, APIService) {
     var ctrl = this;
 
     ctrl.$onInit = function() {
```

Comment:
Not part of this PR but there is a `$scope.namespace` and `$scope.type` right here that seems to be unused.